src/CanvasRenderingContext2d.cc

```diff
@@ -662,10 +663,13 @@ NAN_METHOD(Context2d::PutImageData) {
         uint8_t b = *srcRow++;
         uint8_t a = *srcRow++;
 
-        // argb
-        // performance optimization: fully transparent/opaque pixels can be
-        // processed more efficiently.
-        if (a == 0) {
+        if (dstRow < dstStart) {
```

Comment:
The current npm version is `2.0-alpha.2`, I would like to suggest we publish a `2.0-alpha.3` with the latest code because the exported interface is all changed(BREAKING).

And because it is `alpha` version, I believe it's not a problem.